shared/actions/favorite.js

```diff
@@ -120,6 +124,24 @@ export function favoriteList (): (dispatch: Dispatch) => void {
         const action: FavoriteList = {type: Constants.favoriteList, payload: {folders: folderProps}}
         dispatch(action)
         dispatch(badgeApp('newTLFs', !!(folderProps.publicBadge || folderProps.privateBadge)))
+
+        const newNotify = {public: folderProps.publicBadge, private: folderProps.privateBadge}
+        const total = folderProps.publicBadge + folderProps.privateBadge
+
+        if (total && !_.isEqual(newNotify, previousNotify)) {
+          previousNotify = newNotify
+
+          let body = `You have ${total} new folders`
+
+          if (total <= 3) {
+            console.log(folderProps.private.tlfs, folderProps.public.tlfs)
+            body = [].concat(folderProps.private.tlfs || [], folderProps.public.tlfs || [])
+              .filter(t => t.meta === 'new')
+              .map(t => t.path).join('\n')
+          }
```

Comment:
🐑 🐑 I think it's a little easier to read if the body for more than 3 folders is set in the else statement.